src/components/InfoSection.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styled from "styled-components";
+import {Whatsapp} from '@styled-icons/bootstrap';
+import {Email} from '@styled-icons/material-outlined';
+import {StyledIcon} from '@styled-icons/styled-icon'
+import {Call} from '@styled-icons/fluentui-system-regular';
+import CustomerService from "@assets/new/images/home/customer-service.jpg";
+import Imprint from "@assets/new/images/home/imprint.jpg";
+import QuickChange from "@assets/new/images/home/quickchange.jpg";
+import InfoCard from "./InfoCard";
+
+interface InfoSectionType {
+  image: StaticImageData;
+  description: string;
+  buttons: {
+    link: string;
+    title: string;
+    icon?: StyledIcon;
+  }[];
+}
+[];
+
+const InfoSectionData: InfoSectionType[] = [
+  {
+    image: CustomerService,
+    description:
+      "Our customer service team is available via email, phone call and WhatsApp.",
+    buttons: [
+      { link: "", title: "Call", icon: Call },
+      { link: "", title: "Email", icon: Email },
+      { link: "", title: "Chat", icon: Whatsapp },
+    ],
+  },
+  {
+    image: QuickChange,
+    description: "We are passionate about empowering human potentials.",
+    buttons: [{ link: "", title: "Read more"}],
+  },
+  {
+    image: Imprint,
+    description:
+      "Our materials are represented in hundreds of products across multiple markets in Nigeria.",
+    buttons: [{ link: "", title: "View Products"}],
+  },
+];
+
+function InfoSection() {
+  return (
+    <InfoSectionStyled>
```

Comment:
Ehm, always start with styled. That's the convention and it sounds better.

Reply:
Done